cmd/install/assets/hypershift_operator.go

```diff
@@ -909,6 +907,18 @@ func (o HyperShiftEtcdPriorityClass) Build() *schedulingv1.PriorityClass {
 	}
 }
 
+func HypershiftOperatorPriorityClass() *schedulingv1.PriorityClass {
+	return &schedulingv1.PriorityClass{
+		ObjectMeta: metav1.ObjectMeta{
+			Name: HypershiftOperatorPriortyClass,
+		},
+		// Same as openshift-user-critical, which might not exist on plain kube clusters
+		Value:         1000000000,
```

Comment:
do we want to be the same as `openshift-user-critical` or just use the next priority in our schema `100003000`

Reply:
Updated it to 100003000 and added this to the docs, ptal